src/intentManager.js

```diff
@@ -13,24 +13,27 @@ define([
     $
 ) {
     "use strict";
-    var DEFAULT_XMLNS = "http://opendatakit.org/xforms";
-    function makeODKXIntentTag (form, nodeID, path) {
-        return new ODKXIntentTag(form, {
-            path: path || "",
-            xmlns: DEFAULT_XMLNS,
-            extra: {},
-            response: {},
-            unknownAttributes: {},
-            initialNodeID: nodeID
-        });
-    }
-    function ODKXIntentTag(form, data) {
-        util.BoundPropertyMap.call(this, form, data);
+    var DEFAULT_XMLNS = "http://opendatakit.org/xforms",
+        unmappedIntentTags = {},
```

Comment:
Previously an intent manager (and it's encapsulated state) was created for each new vellum instance. Now it looks like intent tags are stored in a module-level global. What do you think of moving this global state to the plugin's `data` area?